### ISSUE TITLE
Translations: From: and To: too little free space between texts when reading email

### ISSUE THREAD

**Issue by xet7** [ProtonMail/WebClients#329]
Hi,
From: and To: too little free space between texts when reading email:

![protonmain-too-little-space](https://github.com/ProtonMail/WebClients/assets/15545/0ef9928a-8e77-440f-b532-a7dc1f8e44b1)

**Comment by olavinto** [comment]
Yeah, I also noticed this right away and checked the localized strings (they are fine). We also do not have any reasonable way to shorten these (in Finnish) unless we use the English variants or the starting letters (like "L" and "V") but that wouldn't look good at all.

**Comment by nico3333fr** [comment]
Thanks for flagging this :)

We will send a fix as soon as possible.

**Comment by nico3333fr** [comment]
Fix has been developped. Just need a bit of patience for next deploys, and it will be there ;)
